src/App.js

```diff
@@ -1,24 +1,17 @@
 import logo from './logo.svg';
+import Landing from '../src/pages/Landing';
+import Header from '../src/components/Header';
+import Footer from '../src/components/Footer';
 import './App.css';
 
+
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+    <Header/>
+<Landing/>
+<Footer/>
+</>
```

Comment:
- Formatting is really important, make sure you have eslint and/or prettier installed, then enable the format file on save on your vscode, this will make sure everything is formatted. On our projects we don't have problems like that because we can't commit anything while we have lint problems.